docs/manpages/zone2ldap.1.md

```diff
@@ -20,6 +22,15 @@ inserted to an LDAP server.
 --basedn=*DN*
 :    Base DN to store objects below
 
+--create-ent
```

Comment:
Well, it's not necessarily only used for DNSSEC zone, but it's mandatory for them. My main concern would be that renaming this option for '--dnssec' may lead users to believe that only this option is necessary to ready a zone for DNSSEC usage, while this is not true (--domainid must also be set, and I don't want to imply --domainid when --create-ent is in use).